projects/hslayers/src/components/layermanager/layermanager.service.ts

```diff
@@ -810,6 +810,7 @@ export class HsLayerManagerService {
       });
     } else if (this.HsUtilsService.instOf(olLayer, ImageLayer)) {
       source.on('imageloadstart', (event) => {
+        loadProgress.error = false;
```

Comment:
Error should probably be cleared on successful loadend event.

Reply:
@raitisbe Where in code does that occur?

Reply:
> The layers param problem is already fixed in my branch about maxResolution. I should probably merge that
> […](#)
> On Sat, 5 Feb 2022, 13:33 Dailis, ***@***.***> wrote: There are some issues still with sublayers... — Reply to this email directly, view it on GitHub <[#2646 (comment)](https://github.com/hslayers/hslayers-ng/pull/2646#issuecomment-1030605219)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AADAR6UAM35GNWXZOFCXY4TUZUDJLANCNFSM5NS3NHQQ> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>. You are receiving this because you were mentioned.Message ID: ***@***.***>

Still having problems with copied layer sublayers and get caps parsing

Reply:
https://github.com/hslayers/hslayers-ng/blob/aaee0bca9e0a350823589d3fa7b8c0fc1413f151/projects/hslayers/src/components/layermanager/layermanager.service.ts#L818

and 

https://github.com/hslayers/hslayers-ng/blob/aaee0bca9e0a350823589d3fa7b8c0fc1413f151/projects/hslayers/src/components/layermanager/layermanager.service.ts#L832